android/java/org/chromium/chrome/browser/BraveLocalState.java

```diff
@@ -0,0 +1,35 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser;
+
+import org.chromium.base.annotations.JNINamespace;
+import org.chromium.base.annotations.NativeMethods;
+import org.chromium.components.prefs.PrefService;
+
+/**
+ * Helper for retrieving a {@link PrefService} from a browser local state.
+ */
+@JNINamespace("chrome::android")
+public class BraveLocalState {
+    /** Returns the {@link PrefService} associated with the browser's local state */
+    public static PrefService get() {
+        return BraveLocalStateJni.get().getPrefService();
+    }
+
+    /** Makes local state data be written to the disk asap */
```

Comment:
You should not do this, the actual writing of prefs to disk should be left to the pref service